test/cookbooks/test/recipes/default.rb

```diff
@@ -60,6 +60,10 @@
   command 'uname'
 end
 
+def method_defined_outside_choregraphie_context
+  # we do nothing, just want to check we can be called
```

Comment:
where do we actually check this?

Reply:
we dont explicitely check, the fact that it does not fail is sufficient